app/views/local_transaction/search.html.erb

```diff
@@ -11,7 +11,7 @@
   edition: @edition,
 } do %>
 
-  <%= render 'govuk_publishing_components/components/intervention' if should_show_sab_intervention? %>
+  <%= render 'govuk_publishing_components/components/intervention', { suggestion_text: "Something" } if should_show_sab_intervention? %>
```

Comment:
Is the `Something` shown to users? Where did this content come from?

Reply:
Nothing is shown because `should_show_sab_intervention` is never set. This change is a quick fix to get the tests to pass, but the plan is to adapt most of the `sab` code for the next experiment. Then, `Something` will be replaced by something the user will see.